### ISSUE TITLE
Redundant aria-label attribute provided on icons/images that decorate associated text

### ISSUE THREAD

**Issue by zbrownell** [adobe/coral-spectrum#25]
### Expected Behavior

When an icon is provided as a decoration to textual content, then the `aria-label` value should not be set on this image/icon element. Instead, the `aria-hidden` attribute should be added/set to `true` and the `alt` attribute should be added/set to an empty String.

This is the recommended approach to handling visual elements that merely decorate associated text. Otherwise, it becomes redundant for folks to hear an icon description that is nearly identical to the text content that goes with it (e.g. - tab labels, buttons, etc)

### Actual Behavior

In many cases, the `aria-label` is added and a default value is assigned (not localized; separate issue) that is - often - nearly identical in nature to the associated text (e.g. - `add` icon next to text that says "Add users" will have `aria-label` of `add`)

Here is a list of all of the places I've seen this behavior occur, along with a description of the specific problem for each component:

1. `*Button`  - in all button variants (e.g. - AnchorButton, Button, etc) icon has alt property with no value, but aria-hidden is not set to true when text content also exists
2. `Tab`  - icon has aria-label value set, even when label text exists (not icon only tab, so icon labeling is redundant)
3. `SideNavItem` - icon has aria-label value set, even when nav item text exists
4. `ListItem` - icon has aria-label value set, even when list item text exists
5. `ColumnViewItem` - icon has aria-label even when ColumnViewItemContent (text) exists in same column item

### Reproduce Scenario (including but not limited to)

Observe tab labels and icons on the following page:

https://opensource.adobe.com/coral-spectrum/dist/playground/#?livereload=true&screen=vertical&code=MTczLDE0NSw2MSw3OSwxOTUsNDgsMTYsMTM0LDEwMywyNTAsNDMsMTQsMjM5LDE0MSwyNDcsMjAyLDEzNywxMzIsMTAsMTE1LDE0NSwyMTgsMTMzLDIwOSwxNzcsMTQzLDE5OCwxNzMsMTkxLDIyOCwxMTUsMTM0LDI0MCwyMzUsMTM3LDEwOSw4NSwyMCwyMiwyNCwxNTIsMTI0LDMxLDE3NSwyMjMsMTg3LDcxLDM5LDMwLDE1OSwxNSwyNTEsMjExLDIxOSwyMzUsMTEsNzYsMjE3LDIxNyw5NywzNSwxMTAsMTUsNzQsNjEsMTA4LDAsMTMyLDUzLDI1NCwxMCw5LDEwOSwyMDcsNDAsNDcsMjIsMTA1LDY2LDIwNCwxMiwxNjYsMTMyLDIzOSw2MSwxNTUsMTE0LDE0MiwxODAsMjI3LDYwLDY4LDI0NCwyMCwyMzAsMTY0LDE3NiwxNDcsNTgsMTQwLDIxNiwxNjksMjI0LDE4NCwxMCw3MywyMTgsNDUsNjksODQsNTcsMjA1LDE0MiwxMDcsNjcsMTUzLDcxLDQzLDE1MSwxMTUsMTAsMTc5LDIxNSw0NSw4Nyw2OCw3NywyMTUsMTczLDE3LDIyNywxMTcsMzIsMTY5LDEwMCw5OCw2LDc0LDIzNCwzMSw2LDkyLDExMCwyNTQsMjMsOTgsMTMxLDIyNCwyMDUsMTIzLDE5NywyMjcsMTQxLDc5LDE0MCw2NSw0NywxNjAsMTcyLDM2LDIzNCw4OSwyNDMsMjE5LDkwLDExNSwxNTgsNTAsMTcxLDE3MSwxODAsNzQsMTUwLDE2Myw5MywxODksMTI4LDIwNCw3LDI0NiwyMzYsODgsOTEsMjQ3LDIwNSwyMjUsMzYsNzEsNTYsMTIwLDIwLDI1MiwxNzEsMjQ0LDY3LDIsNzAsNSw5NSwzOCw1NiwxMzUsMTI2LDUzLDIyMywxODMsMjI0LDI0NywzMSw4MiwyMzUsMTc4LDI0NywxNTksMTAwLDc5LDkwLDEyNyw4Myw2MiwyMjAsMTAxLDUsMTYwLDk2LDIzLDIyMiwxMzgsOTUsMTc1LDI1Miw5&

#### Steps to Reproduce

(see above)

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)

Chrome Version 79.0.3945.130 (Official Build) (64-bit) MacOS

#### Coral Spectrum version

Latest (see link to playground; above)

#### Sample Code that illustrates the problem (use the Playground if possible)

See link (above) in Reproduce Scenario section

#### Screenshots (if applicable)

N/A

**Comment by CezCz** [comment]
This is a duplicate of 
https://github.com/adobe/coral-spectrum/issues/23

I'll create a pull request with a proposed solution soon.
